requirements.txt

```diff
@@ -1,11 +1,11 @@
-numpy>=1.17
+numpy>=2.0
```

Comment:
Don't do this.  This is why py3.8 doesn't work.  We also don't want to require this (at least not yet), since lots of packages don't work yet with it.